parts/linux/cloud-init/artifacts/ci-syslog-watcher.path

```diff
@@ -0,0 +1,9 @@
+[Unit]
+Description=Monitor the CI syslog status file for changes
+
+[Path]
+PathModified=/var/run/mdsd/update.status
```

Comment:
is that Path Modified trigger the watcher? if yes, what and all cases this being triggered, meaning file create and update etc. ?

Reply:
yes https://www.freedesktop.org/software/systemd/man/systemd.path.html